apps/src/aichat/views/AichatView.tsx

```diff
@@ -71,9 +78,17 @@ const AichatView: React.FunctionComponent = () => {
     );
   }, [dispatch, initialSources, levelAichatSettings]);
 
-  const {botName} = useAppSelector(
-    state => state.aichat.currentAiCustomizations.modelCardInfo
-  );
+  // Showing presentation view when:
+  // 1) levelbuilder hasn't explicitly configured the toggle to be hidden, and
+  // 2) we have a published model card (either by the student, or in readonly form from the levelbuilder)
+  const showPresentationToggle = () => {
+    return (
+      !levelAichatSettings?.hidePresentationPanel &&
+      (hasPublished ||
+        (levelAichatSettings?.visibilities &&
+          isDisabled(levelAichatSettings.visibilities.modelCardInfo)))
```

Comment:
Not sure if this is P0 - If levelbuilders mark visibility of `modelCardInfo` as `readonly`, then it is assumed that all of the fields in model card should be filled out. Maybe worth checking with Dan on this assumption or do levelbuilders need a reminder about this.

Reply:
If this is a P0, we can add a levelbuilder-side check for this as well

Reply:
I already did it, so I'll leave this for now (I think its a sensible default). We can clarify the finer points with levelbuilders / validate down the road